src/modules/common/components/__tests__/__snapshots__/Headroom.spec.js.snap

```diff
@@ -14,7 +14,7 @@ exports[`Headroom should render correct if state is pinned, transition 1`] = `
     translateY={0}
   >
     <div
-      className="Headroomstyles__HeaderWrapper-iDCIAq fsTJoL"
+      className="Headroomstyles__HeaderWrapper-jYoiGy kNRfRu"
```

Comment:
Why are these classnames messed up ?

Reply:
My guess is that they changed the hashing algorithm :P